README.md

```diff
@@ -159,30 +159,29 @@ If you can simplify or improve the installation instructions or add instructions
 The default behavior of `elm-format`-approved plugins is to format Elm files on save.
 
 
-### atom-elm-format installation
-
-1. Install elm-format
-1. Install atom-elm-format
-
-    ```
-    apm install elm-format
-    ```
-
-  or use the Atom package manager in Atom's settings
+### Atom installation
 
+##### Pre-install:
+1. Download the appropriate elm-format binary
+2. Moving it to `usr/local/bin` will save some time later, although you can technically place it anywhere in your file system.
 
-### atom-beautify installation
-
+##### Install:
 1. Install elm-format
-1. Install atom-beautify
+2. Install atom-beautify
+```
+$ apm install elm-format
```

Comment:
Atom-Beautify does not require `elm-format` from `apm` to be installed. It does require `elm-format` binary to be installed, which is already step 1.